cli/cmd/plugin/ui/web/tanzu-ui/node-server/src/routes/api/endpoints/vsphere.js

```diff
@@ -158,16 +158,24 @@ router.get('/resourcepools', (req, res) => {
  * Mock route for getting VC os images
  */
 router.get('/osimages', (req, res) => {
-    winston.info('Mock UI FETCH DATACENTERS (#' + mockOsImageRequestCounter + ')');
-    let osImageResponse = [];
-    if (mockOsImageRequestCounter > 0) {
-        osImageResponse = readFile('provider-vsphere-osimages.json');
+    const dc = '' + req?.query?.dc;
```

Comment:
do we need this empty string?

Reply:
I believe so, because otherwise `dc` can be undefined

Reply:
I removed it; I like it better without and whatever problem I was having before isn't there :)